both/helpers.js

```diff
@@ -32,13 +37,13 @@ register("competition", function(attribute) {
 });
 
 register("roundEventCode", function() {
-  return getRoundAttribute(this.roundId, 'eventCode');
+  return getRound(this.roundId).eventCode;
```

Comment:
I'm happy to get rid of them, but I haven't really thought about what to replace them with. Don't know if there is a general simple pattern, or if it requires a rethink for each usage.

I have been removing them one instance at a time.